ntripclient.c

```diff
@@ -584,6 +590,25 @@ int main(int argc, char **argv)
   }
 #endif
 
+  // setup reading from stdin (IPC pipe) to non-blocking
+  int flags = fcntl(STDIN_FD, F_GETFL);
+  fcntl(STDIN_FD, F_SETFL, flags | O_NONBLOCK);
+
+#ifdef NTRIPCLIENT_DEBUG_LOG
+  FILE *flogfile = fopen("/home/debian/ntripclient/ntripclient.log", "w+");
+  if (!flogfile) {
+    return 20;
+  }
+
+  int idx = 0;
+  fprintf(flogfile, "args...\n");
+  for (idx = 0; idx < argc; idx++) {
```

Comment:
Can scope of `idx` be reduced to inside the for loop? e.g. declaration `for (int idx = 0; ...)`

Reply:
Done!